examples/ping/src/lib.rs

```diff
@@ -21,10 +21,5 @@ pub unsafe extern "C" fn handle() {
     }
 }
 
-#[no_mangle]
-pub unsafe extern "C" fn handle_reply() {
-    msg::reply("PONG", 20_000_000, 0);
-}
-
```

Comment:
Isn't this reply used by some other program?

Reply:
As I can see nobody use this function in tests. So, I think if somebody want to use it somewhere, then he should implement it.